Applications/YOLO/jni/main.cpp

```diff
@@ -254,6 +253,17 @@ int main(int argc, char *argv[]) {
     model->summarize(std::cout, ML_TRAIN_SUMMARY_MODEL);
 
     model->train();
+
+    // // simple inference example
```

Comment:
Is there specific reason to leave this code as comment?

Reply:
It was simply a reference code for inference using dummy data. I removed these codes. thanks!